conbench/templates/compare-list.html

```diff
@@ -55,59 +55,44 @@ <h5 class="card-title"></h5>
-              {% if loop.index == 1 %}
-                <li data-target="#bokeh-carousel" data-slide-to="0" class="active"></li>
-              {% else %}
-                <li data-target="#bokeh-carousel" data-slide-to="{{ loop.index - 1 }}"></li>
-              {% endif %}
-            {% endfor %}
-          {% endif %}
-        </ol>
-        {% for plot in plot_history %}
-          <div class="item  {% if loop.index == 1 %}active{% endif %} ">
-            <div align="center">
-              <a href="{{ outlier_urls[loop.index - 1] }}">{{ outlier_names[loop.index - 1] }}</a>
-            </div>
-            <div id="plot-history-{{ loop.index - 1 }}" align="center"></div>
-          </div>
-        {% endfor %}
-        {% if plot_history|length > 1 %}
-          <br>
-          <br>
-          <br>
-        {% endif %}
-      </div>
-    </div>
+<div class="collapse mb-3" id="collapseHelpblock">
+  <div class="card card-body">
+    <p>
+      Each row in the table below represents a comparison. The outcome of that comparison is summarized by two metrics: <code>change</code> and <code>z-score</code>.
+    </p>
+    <p>
+      There are no definite guidelines for interpretation. However, a
+      rather ignorant rule of thumb is that a <code>z-score</code> value of <code>-5</code> or smaller may indicate an actual performance regression
+      and should lead to brief human review.
+      A good method for such review is to click the link in the corresponding row; it leads to a visual representation of
+      the contender benchmark result, the baseline benchmark result, and the comparable historical ensemble.
+      That type of visualization usually allows a human to assess how significant a change really is.
+    </p>
+    <p>
+      The <code>change</code> column compares two matching benchmark results and expresses the difference in percent.
+    </p>
+    <p>
+      The <code>z-score</code> column shows how the contender benchmark result performs in view of a larger historical ensemble of comparable
+      benchmark results (in this case, the reference run contains only the newest of the results in that ensemble;
+      all other results were submitted via different CI runs in the past).
+    </p>
+    <p>
+      Whether the contender does better or worse than the reference is indicated by sign:
+      a negative sign (for both, <code>change</code> and <code>z-score</code>) reflects the direction of degradation, a positive sign reflects the direction of improvement.
+      The directionality is derived from the user-given unit (units other than s, i/s, B/s are <a href="https://github.com/conbench/conbench/issues/1335">not yet well supported</a>).
+    </p>
+    <p>
+      Further methodological details are described in the
+      <a href="https://github.com/conbench/conbench/blob/52cf34f96cb471f493dc201b0a57ece6132eb10d/docs/pages/lookback_zscore.md">lookback z-score method documentation</a>.
+    </p>
```

Comment:
Right now this reads like this:

![image](https://github.com/conbench/conbench/assets/265630/7f8c9d24-2555-4430-a0d5-ec780ddc0ac7)


Happy to tweak further. Raises questions about external docs vs in-UI docs. 

I think a bit of docs/text/help is required for non-expert users to not be completely lost on this page. Thanks to @thisisnic for providing a bit of feedback, more is always appreciated! :)

Reply:
More feedback it is then! :grin: 

When you talk about units, please could you spell out what the are, e.g. "seconds (s)" and the same for the others.  Without looking it up, I wouldn't know what i/s or B/s is.  Given that previous versions of the benchmarks only had duration metrics, I reckon you also may need to spell out what these other units are and what they represent.

Reply:
> When you talk about units, please could you spell out what the are, e.g. "seconds (s)" and the same for the others.

This is good feedback, and I'd totally agree in most contexts! I carried this to https://github.com/conbench/conbench/issues/1335#issuecomment-1611197697; which is about _properly_ associating a short version of a unit to a long version.

Here I think it's OK to move on for now. What's [hard-coded as of today](https://github.com/conbench/conbench/blob/f1815a0c99211b1df2407f5f96fe069e0d5ddf89/conbench/entities/history.py#L696C17-L696C29) is literally `"B/s", "i/s"`. As a benchmark author one can use these short versions and really mean anything. For example, I assumed that `i/s` is iterations per second, but just now I saw another piece of code that talks about "items per second" (see below). Italians per salad or instructions per supervisor -- all that is confusing, but it boils down to _today_ both of `B/s` and `i/s` are interpreted as "more is better", all other units are interpreted as "less is better". :shrug:.

That madness must be resolved first, and then this tiny help text that we talk about here does not even need to go into these details.

> Given that previous versions of the benchmarks only had duration metrics

Found https://github.com/voltrondata-labs/benchmarks/commit/ff43a5fced084dc73526578a2f8384b6ce16747c#diff-75f2979db7996df563791d136dd2305730e347e4e4624ef8213c922da64eda64R174 and saw that already more than two years ago we had Arrow benchmarks reporting with the unit `i/s` (what the Conbench API sees) and meaning items per second (as we see in the client code).